.gitignore

```diff
@@ -1,8 +1,12 @@
 .idea
-.gradle/
 build/
 out/
 target/
+gradle/
```

Comment:
Can you fix the Travis failure?

Here is the log: https://travis-ci.org/google/data-portability/jobs/334559758

Can you also remove the Maven test from Travis? There's a TODO in https://github.com/google/data-portability/blob/master/.travis.yml, you should just need to remove the language: java line (and its parameters below)

Reply:
Done. How can I test?

Reply:
Do you see in the PR where it says 

continuous-integration/travis-ci/pr — The Travis CI build could not complete due to an error

Click on details there, that should link you to the Travis build. Right now the "matrix" build has 3 parallel builds, 2 Java (they unfortunately say "No language specified" due to a known Travis bug) and 1 Node JS. After the change there should only be 1 Java instead of 2, and if you click the log it should have built with gradle.